composer.json

```diff
@@ -18,11 +18,11 @@
     },
     "require-dev": {
         "composer/installers": "^2",
-        "drupal/core-composer-scaffold": "^10.3.6",
-        "drupal/core-dev": "^10.3.6",
-        "drupal/core-recommended": "^10.3.6",
+        "drupal/core-composer-scaffold": "^10 | ^11@dev",
```

Comment:
How come the @ dev here?

Reply:
Good question, not sure where this is from. Let's try regular release.